src/types.rs

```diff
@@ -131,6 +131,8 @@ pub struct LocalUtxo {
     pub txout: TxOut,
     /// Type of keychain
     pub keychain: KeychainKind,
+    /// This UTXO is used as an input of a mempool tx
```

Comment:
I think a simpler line could be "Whether this UTXO is spent by an unconfirmed tx".